examples/fungible/README.md

```diff
@@ -66,7 +66,7 @@ bytecode:
 (cd examples/fungible && cargo build --release)
 
 BYTECODE_ID=$(linera publish-bytecode \
-    examples/target/wasm32-unknown-unknown/release/fungible_{contract,service}.wasm)
+    ~/linera-protocol/examples/target/wasm32-unknown-unknown/release/fungible_{contract,service}.wasm)
```

Comment:
The line just before contains `cd examples/fungible` so it's clear that we must be at the root of the repository.

Reply:
Also, this change would only work for users who have `linera-protocol` directly in the root of their home directory, and break it for others.